tutor/src/components/notes/summary-page.jsx

```diff
@@ -80,6 +80,9 @@ class NoteSummaryPage extends React.Component {
 
   @action.bound onDelete(note) {
     this.props.onDelete(note);
+    this.selectedPages.remove(
+      this.selectedPages.find(s => s.uuid == note.page.uuid)
+    );
```

Comment:
As a side effect this would deselect a section that still contains notes, I think?

Reply:
no, the selection is held in the component in the `selectedPages` array.  I had to remove it from there as well otherwise it'd never show the "please select some sections" message